map.css

```diff
@@ -2,3 +2,79 @@
   height: 40vw;
   width: 100%;
 }
+
+#searchMap {
+  height: 40vw;
+  width: 100%;
+}
+
+#description {
+  font-family: Roboto;
```

Comment:
i think this should be set for either whole `body` or some specific classes, not just one element. let's stick to one font family on the site